### PR TITLE
search input auto focus

### PR DESCRIPTION
**Pull request by ItayRaz**

### CODE REVIEW

File: src/Common/FilterSelectBox/SearchInput.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ function SearchInput({ onChange, searchTerm, delay = 1000, placeholder }, ref) {
         onChange(value);
     }, delay);
 
+    if (inputEl.current) {
```
Comment (avipunes) [inline src/Common/FilterSelectBox/SearchInput.js:13]: this will always focus the input...

**Comment by avipunes** [comment]
@ItayRaz please add a link to jira story in the PR desc.

File: src/Common/FilterSelectBox/SearchInput.js (hunk 1 above)
Comment (avipunes) [reply src/Common/FilterSelectBox/SearchInput.js:13]: every time we render the component.

Comment (ItayRaz) [reply src/Common/FilterSelectBox/SearchInput.js:13]: Is it a bad thing? When the cmp is open the input is always focus and i think it's ok. instead if clicking the input when you want to search, you can just type

Comment (avipunes) [reply src/Common/FilterSelectBox/SearchInput.js:13]: this might create bugs, but let's see that in action.

**Comment by avipunes** [comment]
@ItayRaz you need to upgrade the version.

**Pull request merged**